tests/repository_simulator.py

```diff
@@ -274,6 +286,7 @@ def _compute_hashes_and_length(
         return hashes, len(data)
 
     def update_timestamp(self):
+        """Update timestamp and assign snapshot ver to snapshot_meta ver"""
```

Comment:
Add the snapshot_meta between simple quotes. I would suggest being explicit and using version instead ver.

Reply:
I used "ver" instead of "version" in order to keep it a one-line docstring as this is a short function, but if it's unclear will change it.

Reply:
It is clear, but IMHO, I would pick up the exact reference in the documentation between a one-line docstring and an explicit reference. 
I still see my comment as optional as it is test documentation.